test/test_artifacts/v1/autogluon.test.Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM $COSMOS_IMAGE as base
 
 ARG MAMBA_DOCKERFILE_ACTIVATE=1
 
-RUN micromamba install --freeze-installed -y conda-forge::pytest
+RUN micromamba install --freeze-installed -y -c conda-forge pytest
```

Comment:
In this version, pytest is now installed by default; I found that installing `conda-forge::pytest` leads to error due to existing installation, whereas `-c conda-forge pytest` skips existing installation.